Documentation/simmodelerCAD-workflow.rst

```diff
@@ -0,0 +1,141 @@
+------------
+See :ref:`SimModeler prerequisite`.
+
+We use scripts available `here <https://github.com/SeisSol/Meshing/blob/master/GocadRelatedScripts>`__ to generate surface meshes of the faults.
+The dataset for generating the Palu structural model is available `here <https://github.com/SeisSol/Meshing/blob/master/GocadRelatedScripts/ExampleFiles/SimModeler_workflow/>`__.
+
+Creating the topographic layer
+------------------------------
+
+We create the topography from a netcdf file downloaded from https://www.gebco.net/
+The domain range from longitude 118.9 to 121.7 and from latitude -2.4 to 1.0.
+We then project the data (see :ref:`On the use of projections` for the choice of a projection), triangulate it, and export it as stl (list of triangles) using:
+
+.. code-block:: bash
+
+   python SeisSol/Meshing/GocadRelatedScripts/createGOCADTSurfNXNY_netcdf.py --proj '+init=EPSG:23839' data/GEBCO_2014_2D_118.1904_-2.4353_121.6855_1.0113.nc bathy.stl
+
+We then load the stl file into SimModeler
+
+.. code-block:: none
+
+    File>import discrete data>bathy.stl
+
+The option ``Find Edges by Face Normal`` allows isolating groups of triangles using the relative angle between their face normals.
+If the angle value is small, the imported surface will appear as divided into many faces.
+These faces must then be explicitly accounted for by the mesh. If some faces are tiny, this will mechanically lead to small cells in the final mesh.
+We, therefore, recommand to use a large enough value.
+
+
+Creating the domain box
+-----------------------
+
+We generate a surface mesh of a simple box using gmsh:
+
+.. code-block:: bash
+
+   gmsh -2 create_box.geo -format stl
+
+The box dimensions are such as the topography is slightly wider than the box.
+The mesh size is chosen small enough to facilitate intersection with topography and large enough to limit the number of elements.
+
+.. code-block:: none
+
+    mesh_size = 10e3;
+    Xmax = -160e3;
+    Xmin = 215e3;
+    Ymin = 1235e3;
+    Ymax = 1605e3;
+    Zmin=-200e3;
+    Zmax=5e3;
+
+
+Creating faults
+---------------
+
+We use `this script <https://github.com/SeisSol/Meshing/blob/master/GocadRelatedScripts/createFaultFromCurve.py>`__ to generate surface meshes of the faults, using inferred fault traces and dip description.
+The scripts first resample the 2D fault trace and then can smooth them.
+Finally, the smoothed and resampled traces are swept towards negative and positive (if the topography has positive elevation) z.
+
+createFaultFromCurve.py takes 3 main arguments: ``filename``, ``dipType`` and ``dipDesc``.  
+
+- ``filename`` is the name of the ASCII file describing the trace.  
+- ``dipType`` allow switching between a constant (0), an along-depth dependant (1) or an along-strike dependent (2).
+- ``dipDesc`` gives either the dip angle value (dipType=0) or the 1D variation of the dip angle (dipType=1 or 2).
+
+.. figure:: https://www.geophysik.uni-muenchen.de/~ulrich/dipDescription.png
```

Comment:
should we move all image of the doc to imgur?

Reply:
Why should we move it there?

Reply:
because we don't want large files on a git repository?
And because if someday I leave LMU and my personal account gets closed, the images won't be available anymore?

Reply:
You are right, but we have already some pictures in the repo, see Documentation/LatexFigures 
I guess https://git-lfs.github.com/ might be interesting.

Reply:
I know for Documentation/LatexFigures, this wouldn't have append if we would have merged the cookbook through pull request (anyway, the files are not extremely large).
https://git-lfs.github.com/ looks like a good option.

Reply:
So I suppose you check in your pictures to the git and then we do a git lfs migrate afterwards.